spec/models/grace_period_submission_rule_spec.rb

```diff
@@ -429,28 +432,29 @@ class GracePeriodSubmissionRuleTest < ActiveSupport::TestCase
           # Assert that each accepted member of this grouping did not get a GracePeriodDeduction
           @grouping.reload
           @grouping.accepted_student_memberships.each do |student_membership|
-            assert_equal members[student_membership.user.id] -1, student_membership.user.remaining_grace_credits
+            expect(student_membership.user.remaining_grace_credits).to eq(members[student_membership.user.id]-1)
           end
 
-          # We should have all files except OvertimeFile1.java and NotIncluded.java in the repository.
-          assert_not_nil submission.submission_files.find_by_filename('TestFile.java')
-          assert_not_nil submission.submission_files.find_by_filename('Test.java')
-          assert_not_nil submission.submission_files.find_by_filename('Driver.java')
-          assert_not_nil submission.submission_files.find_by_filename('OvertimeFile1.java')
-          assert_nil submission.submission_files.find_by_filename('NotIncluded.java')
-          assert_not_nil submission.get_latest_result
+          # We should have all files except NotIncluded.java in the repository.
+          expect(submission.submission_files.find_by_filename('TestFile.java')).not_to be_nil
+          expect(submission.submission_files.find_by_filename('Test.java')).not_to be_nil
+          expect(submission.submission_files.find_by_filename('Driver.java')).not_to be_nil
+          expect(submission.submission_files.find_by_filename('OvertimeFile1.java')).not_to be_nil
+          expect(submission.submission_files.find_by_filename('NotIncluded.java')).to be_nil
+          expect(submission.get_latest_result).not_to be_nil
         end
       end
-
     end
   end
 
   private
 
   def submit_files_before_due_date
     pretend_now_is(Time.parse('July 20 2009 5:00PM')) do
-      assert Time.now < @assignment.due_date
-      assert Time.now < @assignment.submission_rule.calculate_collection_time
+      expect(Time.now).to be < @assignment.due_date
```

Comment:
There should be no `expect` statements in helper functions. Instead, write some tests that call this helper and then check that each of your `expect`s pass.  That way, if the helper fails for any reason we see why in exactly one place instead of in every test that calls the helper. 

Also, helper functions should be put in `spec/support/helpers.rb` if they are a generic helper method (could be used by multiple specs).